.gitlab-ci.yml

```diff
@@ -28,12 +28,13 @@ generate-scripts:
       - install_script_vector0.sh
 
 unit_tests:
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
+  image: registry.ddbuild.io/images/mirror/ubuntu:22.04
```

Comment:
I don't really remember why I did this change (but I was the one who created that step so I just changed something I did) most probably because buildimages is a ubuntu 14:04 and I needed something more recent (e.g. python3 for my extract function)

Reply:
I'm all for using a more up to date image, but I'd be in favor of bundling the requirements in that image, not having to install something is likely to save us from sporadic failures and it's still a little time save.
Definitely not a blocking comment though